app/src/main/java/org/inspire/breath/activities/FeverMainQ.java

```diff
@@ -0,0 +1,32 @@
+package org.inspire.breath.activities;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+
+import org.inspire.breath.R;
+
+public class FeverMainQ extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_fever_main_q);
+    }
+
+    {
```

Comment:
This should be contained in a method

Edit: Assuming this is the Yes question, it should invoke the MalariaActivity (or change these activities into fragments and swap to the MalariaFragment)